modules/mesh-task/variables.tf

```diff
@@ -145,7 +145,7 @@ variable "outbound_only" {
 variable "consul_ecs_image" {
   description = "consul-ecs Docker image."
   type        = string
-  default     = "hashicorppreview/consul-ecs:0.8.0-dev"
+  default     = "ganeshrockz/rearch-ecs"
```

Comment:
Do we need this change? The other variable file has this as `default     = "hashicorppreview/consul-ecs:0.8.0-dev"`

Reply:
Yes. Good catch